File: ArrayAndList/missing_number/missing_number.java
package ArrayAndList.missing_number;

public class missing_number {
    public static void main(String[] args) {
        int[] arr={1,2,3,4,6,7};
        int n=arr.length;
        int b=arr[n-1];

        int total=b*(b+1)/2;
        int sum=0;
        for(int i=0;i<n;i++){
            sum=sum+arr[i];
            
        }
        int miss=total-sum;
        System.out.print(miss);

    }
}

File: ArrayAndList/missing_number/missing_number.js
let arr=[1,2,3,4,5,7];
let n=arr.length;
let a=arr[n-1];

let total=a*(a+1)/2;
let sum=0;
for(let i=0;i<n;i++){
    sum=sum+arr[i];
}
let missed=total-sum;
console.log(missed);